flw/src/main/resources/flw.properties

```diff
@@ -0,0 +1,7 @@
+#what time of day should the process of generating the targetFile start
```

Comment:
It doesn't, I just didn't update the comment that I pulled from your config file.